docs/search_algorithm_switching.md

```diff
@@ -1,37 +1,533 @@
+        electronic_portfolio_s,
+        cataloged_tdt,
+        contained_in_s
+      </str>
+
+      <str name="facet">true</str>
+      <str name="facet.mincount">1</str>
+      <str name="facet.limit">10</str>
+      <str name="facet.field">format</str>
+      <str name="facet.field">language_facet</str>
+      <str name="facet.field">pub_date_start_sort</str>
+      <str name="facet.field">advanced_location_s</str>
+      <str name="f.format.facet.sort">index</str>
+      <str name="f.advanced_location_s.facet.sort">index</str>
+      <str name="f.language_facet.facet.limit">1000</str>
+      <str name="f.advanced_location_s.facet.limit">500</str>
+    </lst>
+  </initParams>
+```
+
+In order to further extend the default parameters sent within HTTP requests to
+Solr, one need not directly modify the `initParams`, but instead override by following this pattern:
+
+```xml
+    <lst name="appends">
+      <str name="boost">if(termfreq(text,'engineering'),100,0)</str>
+    </lst>
+```
+
+In this case, `<lst>` specifies the list of parameters to send in the request,
+and the attribute `appends` qualifies the the child elements are to be appended
+to the `<lst>` elements specified within the `initParams`. The `<str>` encodes a parameter used for boosting search results by certain criteria (hence the `name="boost"`). The function query `if(termfreq(text, 'engineering'),100,0)` determines if the `text` field contains the string `engineering`, and if so, boosts the weight of the result by 100. If it does not contain `engineering`, it boosts the weight by 0.For more information, please referencing the [Solr documentation specific to configuring request handlers](https://solr.apache.org/guide/solr/latest/configuration-guide/requesthandlers-searchcomponents.html#defaults-appends-and-invariants).
+
+#### Submitting Proposed Changes
+
+Please proceed at this point with opening a pull request and requesting a review
+of the updated `ConfigSet`. An example of this can be found within the
+following:
+
+- https://github.com/pulibrary/pul_solr/pull/375
+- https://github.com/pulibrary/pul_solr/pull/377
+
+#### Deploying
+
+One must then deploy the updated `SolrConfig` settings using [Capistrano](https://capistranorb.com/) with the following:
+
+```bash
+$ bundle exec cap staging deploy
+$ bundle exec cap staging configsets:update[orangelight-staging/conf,orangeligh-staging-config]"
+```
+
+Please reference the [PUL Solr documentation](https://github.com/pulibrary/pul_solr#managing-configsets) for further guidance regarding the updating of Solr `ConfigSets`.
+
+### Integration with Blacklight
+
+One may integrate support for switching the search algorithms by following the
+reference implementation undertaken by Group 1 of the Search and Race initiative
+for Orangelight:
+
+- https://github.com/pulibrary/orangelight/pull/3813
+- https://github.com/pulibrary/orangelight/pull/3819
+- https://github.com/pulibrary/orangelight/pull/3827
+
```

Comment:
Can we add an href text for these links? Preferable the title of the pull request.